### ISSUE TITLE
SVG logos don't work on all platforms?

### ISSUE THREAD

**Issue by jbednar** [holoviz/panel#2618]
As mentioned in https://github.com/holoviz/panel/pull/2609, on some platforms it seems that at least some Panel logo SVGs show up in Times Roman rather than the Ubuntu font they were designed for. @MarcSkovMadsen, it sounds like you have access to such a platform, so can you please view the following SVGs and check the box if it appears properly on that platform?

- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked.svg
- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked_currentColor.svg
- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked_current_theme.svg
- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked_dark.svg
- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked_dark_theme.svg
- [ ] https://github.com/holoviz/panel/blob/master/doc/_static/wordmark.svg

@philipp, do we need this seemingly out-of-date version?
https://github.com/holoviz/panel/blob/master/doc/_static/logo_stacked_square.svg

**Comment by jbednar** [comment]
Also, I think we should extend the standard logo filenames officially to cover the various cases above.  The [existing list of standard names](https://github.com/pyviz-dev/nbsite/issues/45) doesn't have anything about theming yet.  Presumably "currentColor" should be "current_color" to match the other names, as well.

**Comment by mattpap** [comment]
Ubuntu is not a universal font, so the behavior here is correct (falling back to the default font). If you want to keep the specific font and make things platform independent, then given this is a one word logo, I would convert the text node into a path and sever reliance on text rendering altogether.

**Comment by jbednar** [comment]
Right; just need to find out which (or is it all?) logo files have not been converted in that way.

**Comment by mattpap** [comment]
It's all (relevant). Converting text to paths increases SVG size by about 30% (still insignificant compared to PNG).
